packages/ui/client/components/explorer/ExplorerItem.vue

```diff
@@ -219,12 +238,12 @@ const projectNameTextColor = computed(() => {
       </VueTooltip>
       <IconButton
         v-if="!isReport"
-        v-tooltip.bottom="'Run current test'"
+        v-tooltip.bottom="runButtonTitle"
         data-testid="btn-run-test"
-        title="Run current test"
+        :title="runButtonTitle"
         icon="i-carbon:play-filled-alt"
         text-green5
-        :disabled="type !== 'file'"
+        :disabled="state === 'todo'"
```

Comment:
review this including `standalone` check, we shouldn't allow run skipped tests (check `ui.spec.ts` and the `standalone` suite: all files when using `standalone` initialized with `state` set to `skip`)

Reply:
we should ignore any state, `disabled` should never be `true` here